README.md

```diff
@@ -390,15 +416,12 @@ allowing further manipulation by script.
 as well as the options for this particular showing of the toast.
 Thus, the possible options are:
 
-- `type`, like the attribute
-- `position`, like the attribute
+- `type`, like the property that reflects the `type=""` attribute
```

Comment:
Is it convention to write these as "`type=""` attribute" or would just "`type` attribute" work?

Reply:
including the `=""` makes it more obvious. Especially because, for historical reasons, specs talk about "IDL attributes" as a different way of saying "JS property", so when you say "`type` attribute" it's a bit ambiguous.